.github/pull_request_template.md

```diff
@@ -1,8 +1,6 @@
-- [ ] Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below
-
 **End-user friendly description of the problem this fixes or functionality that this introduces**
 
-
+- [ ] Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below
```

Comment:
maybe just add 2 linebreaks above and then go for merge :)

Reply:
It unfortunately doesn't make a difference in the PR template. Here's how it looks like in this PR:
![Screenshot 2024-10-02 at 11 00 14 AM](https://github.com/user-attachments/assets/fbc09553-a240-46f4-a937-e10b4792d2ec)

Reply:
😮 weird!